### PR TITLE
Improve pie tasks

### PR DESCRIPTION
**Pull request by adamkerz**
Working to make usable to easily spin up different dbs and endpoints for different uses.

### CODE REVIEW

File: pie_docker.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+"""
+Python3.6+ only
+"""
+from pie import *
+
+
+
+class Docker:
+    def __init__(self,options=[]):
```
Comment (koriaf) [inline pie_docker.py:9]: object instance as a parameter

File: pie_docker.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        self.options=options
+
+    def build(self,context,options=[]):
```
Comment (koriaf) [inline pie_docker.py:12]: which is probably not a big deal because it's never called twice, but anyway

**Comment by monkeypants** [comment]
> object instance as a parameter

what do you mean? are you referring to the `class Docker(object):` thing in python 2 that makes classes "new style" (not needed in python 3)?

File: pie_docker.py (hunk 1 above)
Comment (monkeypants) [reply pie_docker.py:9]: > object instance as a parameter

what do you mean? are you referring to the `class Docker(object):` thing in python 2 that makes classes "new style" (not needed in python 3)?

Comment (koriaf) [reply pie_docker.py:9]: https://stackoverflow.com/a/2831215

File: pie_docker.py (hunk 2 above)
Comment (adamkerz) [reply pie_docker.py:12]: Well, it's read only... unless it doesn't become so, I guess :-)

Comment (adamkerz) [reply pie_docker.py:12]: (will fix, thank you)

File: pie_docker.py (hunk 1 above)
Comment (monkeypants) [reply pie_docker.py:9]: Oh! that would have caught me out, I didn't realise.

**Pull request merged**